external_deps/build.sh

```diff
@@ -653,6 +655,10 @@ build_install() {
 		find "${PKG_PREFIX}/lib" -name '*.exp' -execdir rm -f -- {} \;
 		;;
 	esac
+
+	rmdir "${PKG_PREFIX}/bin" 2> /dev/null || true
```

Comment:
Maybe doing it that way would be better?

```sh
[ -d "${PKG_PREFIX}/bin" ] && rmdir "${PKG_PREFIX}/bin"
```

Reply:
Hmmm, wait, it would still return `false` if it does not exist and stop the execution…

Better do the opposite:

```
! [ -d "${PKG_PREFIX}/bin" ] || rmdir "${PKG_PREFIX}/bin"
```

Or also we may just do a reusable function for that (and proper `if` call) to avoid copy pasting.

Reply:
The directory should always exist (thanks to the other lines added in this commit); the idea is to remove it if it's empty.

I see there's an `rmdir` option on my machine `--ignore-failure-on-non-empty` doing exactly this, but it probably doesn't work on Mac or something

Reply:
Ah ok, then you can do that:

```sh
find "${PKG_PREFIX}/bin" "${PKG_PREFIX}/include" "${PKG_PREFIX}/lib" -maxdepth 0 -empty -exec rmdir {} \;
```

I verified it works on macOS (and yes macOS does not support `rmdir --ignore-failure-on-non-empty`).

Reply:
I replaced it with a command to delete all empty subdirectories, which seems like a good idea and is also shorter.

Reply:
Great! 👍